change/@microsoft-teams-js-4e6e7b05-3dd3-4b45-b74c-912281bbea04.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "gitmagic additions to 2.0-preview for March 09, 2022. Changes merged in from V1 include removing private tag for sharing, making Menu APIs Public, and adding new features for files API(enum FileDownloadStatus, interface IFileItem, getFileDownloads(), openDownloadFolder())",
```

Comment:
nit: can you add a space between 'API' and '(' in this part: "files **API(**enum FileDownloadStatus,"